### PR TITLE
refactor(dms/rocketmq_dataSource): fix dms rocketmq data source lint issues 

### PR DESCRIPTION
**Pull request by houpeng80**
**What this PR does / why we need it**:
  fix dms rocketmq data source lint issues 
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
  fix dms rocketmq data source lint issues 
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/dms/' TESTARGS='-run TestAccDatasourceDmsRocketMQBroker_basic' 
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/dms/ -v -run TestAccDatasourceDmsRocketMQBroker_basic -timeout 360m -parallel 4 
=== RUN   TestAccDatasourceDmsRocketMQBroker_basic 
=== PAUSE TestAccDatasourceDmsRocketMQBroker_basic
=== CONT  TestAccDatasourceDmsRocketMQBroker_basic

--- PASS: TestAccDatasourceDmsRocketMQBroker_basic (706.96s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dms       707.013s

make testacc TEST='./huaweicloud/services/acceptance/dms/' TESTARGS='-run TestAccDatasourceDmsRocketMQInstances_basic'  
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/dms/ -v -run TestAccDatasourceDmsRocketMQInstances_basic -timeout 360m -parallel 4 
=== RUN   TestAccDatasourceDmsRocketMQInstances_basic 
=== PAUSE TestAccDatasourceDmsRocketMQInstances_basic 
=== CONT  TestAccDatasourceDmsRocketMQInstances_basic 
--- PASS: TestAccDatasourceDmsRocketMQInstances_basic (675.85s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/dms       675.901s
```

### CODE REVIEW

File: huaweicloud/services/dms/data_source_huaweicloud_dms_rocketmq_instances.go [hunk 1]
```diff
@@ -50,15 +51,15 @@ instance name.`,
...
+				Elem:        RocketMQInstancesInstanceRefSchema(),
 				Computed:    true,
 				Description: `Indicates the list of DMS rocketMQ instances.`,
 			},
 		},
 	}
 }
 
-func DmsRocketMQInstancesInstanceRefSchema() *schema.Resource {
+func RocketMQInstancesInstanceRefSchema() *schema.Resource {
```
Comment (chengxiangdong) [inline huaweicloud/services/dms/data_source_huaweicloud_dms_rocketmq_instances.go]:
```suggestion
func rocketMQInstancesInstanceRefSchema() *schema.Resource {
```
it seems it should be a private function

File: huaweicloud/services/dms/data_source_huaweicloud_dms_rocketmq_instances.go [hunk 2]
```diff
@@ -229,14 +230,14 @@ func DmsRocketMQInstancesInstanceRefSchema() *schema.Resource {
 	return &sc
 }
 
-func DmsRocketMQInstancesInstanceRefCrossVpcInfoRefSchema() *schema.Resource {
+func RocketMQInstancesInstanceRefCrossVpcInfoRefSchema() *schema.Resource {
```
Comment (chengxiangdong) [inline huaweicloud/services/dms/data_source_huaweicloud_dms_rocketmq_instances.go]:
```suggestion
func rocketMQInstancesInstanceRefCrossVpcInfoRefSchema() *schema.Resource {
```
it seems it should be a private function

**Review by chengxiangdong — COMMENTED** [review]
I think it is better to define `"dmsv2"` as a constant, although it is an automatically generated code.

File: huaweicloud/services/dms/data_source_huaweicloud_dms_rocketmq_instances.go (hunk 1 above)
Comment (houpeng80) [reply huaweicloud/services/dms/data_source_huaweicloud_dms_rocketmq_instances.go]: done

File: huaweicloud/services/dms/data_source_huaweicloud_dms_rocketmq_instances.go (hunk 2 above)
Comment (houpeng80) [reply huaweicloud/services/dms/data_source_huaweicloud_dms_rocketmq_instances.go]: done

**Pull request merged**